Lib9c/Addresses.cs

```diff
@@ -30,5 +32,15 @@ public static class Addresses
         public static Address GetSheetAddress(string sheetName) => TableSheet.Derive(sheetName);
 
         public static Address GetItemAddress(Guid itemId) => Blacksmith.Derive(itemId.ToString());
+
+        public static Address GetDailyCrystalCostAddress(int index)
+        {
+            return MaterialCost.Derive($"daily_{index.ToString(CultureInfo.InvariantCulture)}");
```

Comment:
```suggestion
            return MaterialCost.Derive($"daily_{index.ToString()}");
```
I think... there could be problems depending on the user's culture setting if CultureInfo goes in.

Reply:
I used InvariantCultureInfo to be unaffected by the user's environment. https://docs.microsoft.com/ko-kr/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-6.0

Reply:
I misunderstood. Thank you for your answer!